src/lib/sys.c

```diff
@@ -286,6 +286,22 @@ bool sys_file_exists(const char *filename)
 	return false;
 }
 
+bool sys_absolute_path_exists(const char *filename)
+{
+	char *abspath = NULL;
+	bool ret = false;
+
+	abspath = realpath(filename, NULL);
+	if (strcmp(abspath, filename) == 0) {
+		ret = true;
+	}
+
+	if (abspath) {
+		free(abspath);
```

Comment:
free(NULL) works fine. You don't need to check for NULL here.

Reply:
cool, thanks for the tip.